src/app.rs

```diff
@@ -735,4 +743,18 @@ impl App {
             };
         };
     }
+
+    pub fn get_song_is_liked(&mut self, id: String) {
```

Comment:
Could rename this function to `current_user_saved_tracks_contains` and let it accept an vec of ids.

```rust
pub fn current_user_saved_tracks_contains(&mut self, ids: Vec<String>) {
```